Prototypr(EN).md

```diff
@@ -0,0 +1,30 @@
+# Prototype(EN)
+
+## Contract Code
+ [prototype.sol](https://github.com/vechain/thor/blob/master/builtin/gen/prototype.sol)
+
+## Contract Address
+ `0x000000000000000000000050726f746f74797065`
+
+>The above mentioned prototype contract address are same on both the test net and main net.
+
+##  Function
+
+|Function|Description|Parameter|
+|---|---|----|
+|master|By default, the contract deployer is the contract holder. The contract holder's address will be returned as a result of calling contract functions.|self: The method caller's address.|
+|setMaster|Set the new contract holder (Only the current contract holder has privilege to do this operation.)|self: The method caller' address<br>newMaster: New contract's holder's address.|
+|balance|Return the specified block's VET and VTHO balance.|self: The method caller' address.<br>blockNumber: Block number (The range of block number varies from 65535 ahead of latest block number to latest block number.)|
+|energy|Return the specified block's VTHO balance.|self: The method caller's address.<br>blockNumber: Block number (The range of block number varies from 65535 ahead of latest block number to latest block number.)|
+|hasCode|check the account type of the contract caller (contract account/normal account).|self: The method caller' address.|
+|storageFor|The account model's content stored in storage.|self: The method caller' address.<br>key: [Reference](https://solidity.readthedocs.io/en/latest/miscellaneous.html#layout-of-state-variables-in-storage)|
+|userPlan|Return the user planned balance and the recovery rate of plan.|self: The method caller' address.|
+|setUserPlan|Set balance and recovery rate for all contract users.|self: The method caller' address.<br>credit: The available balance set to be used(Wei).<br>recoveryRate: The balance recovery rate based on per second(Wei).|
```

Comment:
Set balance and recovery rate => Set credit and recovery rate
The available balance set to be used => The maximum credit set to be used(Wei)

Reply:
The maximum credit set to be used(Wei) => The maximum balance that can be used(Wei)